sdk/core/azure-core/azure/core/_utils.py

```diff
@@ -0,0 +1,31 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import datetime
+
+
+class _FixedOffset(datetime.tzinfo):
```

Comment:
I would prefer to have a compat shim function for this rather than relying on all consumers to do a conditional import.

Reply:
..but that can be fixed after merge since this is an implementation detail.

Reply:
i'm tracking this here https://github.com/Azure/azure-sdk-for-python/issues/16861